developer/developer-guide.md

```diff
@@ -1,3 +1,872 @@
 \include{metadata.md}
-# PLACEHOLDER
-Test
\ No newline at end of file
+
+# About
+Alfred API abstracts away past and future changes to the Alfresco, across major and minor 
+versions, providing a stable interface to Alfresco on which client-side applications can be built.
+
+It also provides functional grouping of related operations from the Alfresco Public API,
+and additional endpoints that are not supported by the Alfresco Public API.
+ 
+
+# Development
+
+## Rules for pull requests
+* Common sense trumps all rules.
+* For every pull request please extend the [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md).
+* Do not make breaking changes since this is an API used by customers. Breaking changes include 
+  adding, changing or removing endpoints or JSON objects used in requests and responses.
+  * If you are forced to make a breaking change:
+    * Notify maintainers
+    * Add a note to the changelog with upgrade instructions
+    * Notify all customers at the next release
+* When working in REST code, please comply to **REST HTTP result codes** policy outlined in the
+  [user guide](https://docs.xenit.eu/alfred-api/stable-user).
```

Comment:
Needs to become https://docs.xenit.eu/alfred-api/stable-developer/rest-api/index.html#rest-http-result-codes